eventhub/Module.md

```diff
@@ -58,33 +25,44 @@ event hubs. It includes operations to revoke publisher, get revoked publishers,
 
 * An Event Hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hub 
 
+### Obtaining Tokens
 * Connection String of the Event Hub Namespace
-We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
-string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
-instructions given below.
-https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
+
+    We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
+    string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
+    instructions given below.
+    https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
+    (Eg: "Endpoint=sb://<Namespace_Name>.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=i1f2BXAbmtxhu............f9GMyfnNWvQ3CY=")
 
 * Shared Access Signature (SAS) Authentication Credentials
-You need to extract the Shared Access Key Name, Shared Access Key, Resource URI to the Event Hub Namespace separately 
-from the connection string.
-    * Shared Access Key Name, 
-    * Shared Access Key, 
-    * Resource URI to the Event Hub Namespace.
 
-# Quickstart(s):
+    You need to extract the Shared Access Key Name, Shared Access Key, Resource URI to the Event Hub Namespace separately 
+    from the connection string.
+    * Shared Access Key Name, (Eg: "RootManageSharedAccessKey")
+    * Shared Access Key, (Eg: "i1f2BXAbmtxhu............f9GMyfnNWvQ3CY=")
+    * Resource URI to the Event Hub Namespace. (Eg: "<Namespace_Name>.servicebus.windows.net")
+
+* Provide the following configuration information in the `Config.toml` file to use the Azure Event Hub connector.
 
-## Publish Events to an Azure Event Hub 
+    ```ballerina
+    sasKeyName = "<SHARED_ACCESS_KEY_NAME>"
+    sasKey = "<SHARED_ACCESS_KEY>"
+    resourceUri = "<RESOURCE_URI_TO_EVENTHUB_NAMESPACE>" 
+    ```
+## Quickstart(s)
+
+### Publish Events to an Azure Event Hub 
 
 This is the simplest scenario to send events to an Azure Event Hub. You need to obtain a connection string of the 
 name space of the event hub you want to send events. 
 
-### Step 1: Import the Azure Event Hub Ballerina Library
+#### Step 1: Import the Azure Event Hub Ballerina Library
 First, import the ballerinax/azure_eventhub module into the Ballerina project.
 ```ballerina
     import ballerinax/azure_eventhub;
 ```
 
-### Step 2: Initialize the Azure Event Hub PublisherClient
+#### Step 2: Initialize the Azure Event Hub PublisherClient
 You can now make the connection configuration using the shared access key name, shared access key, and the resource 
```

Comment:
Use the extracted shared access key name, shared access key, and the resource uri to initialize the Azure Event Hub client to publish data?

Reply:
Fixed in df7c75b . Thank you.